Formula/cfn-format.rb

```diff
@@ -0,0 +1,23 @@
+class CfnFormat < Formula
+  desc "Command-line tool for formatting AWS CloudFormation templates"
+  homepage "https://github.com/aws-cloudformation/rain"
+  url "https://github.com/aws-cloudformation/rain/archive/v1.0.2.tar.gz"
+  sha256 "09bb8dd9a17648ecd7735d75e701c00aebfa07508c1e89043524911e805d079e"
+  license "Apache-2.0"
+  revision 1
```

Comment:
It shouldn't have a revision as a new formula.

Reply:
Whoops. Fixed.